src/commands/aws-accounts.js

```diff
@@ -0,0 +1,89 @@
+const { Command, flags } = require("@oclif/command");
+const AWSCredentials = require("manage-aws-credentials");
+const spawn = require("child_process").spawn;
+const path = require("path");
+const os = require("os");
+
+class AWSAccounts extends Command {
+  static args = [
+    {
+      name: "action",
+      required: true,
+      hidden: false,
+      options: ["add", "remove", "show"],
+      description: "action that will do the command",
+    },
+  ];
+
+  async run() {
+    const { flags, args } = this.parse(AWSAccounts);
+    const name = flags.name || undefined;
+    const access_key = flags.key || undefined;
+    const secret_access_key = flags.secret || undefined;
+
+    switch (args.action) {
+      case "add": {
+        if (name && access_key && secret_access_key) {
+          AWSCredentials.add_profile(name, {
+            access_key,
+            secret_access_key,
+          });
+          const fileSaved = AWSCredentials.save_file();
+          this.log(
+            fileSaved === 1
+              ? "Account successfully saved"
+              : "There were an issue saving your new account"
+          );
+          break;
+        }
+        throw new Error(
+          `The flags name, key, and secret are required, use the flag --help for documentation`
+        );
+      }
+      case "remove": {
+        if (name) {
+          AWSCredentials.delete_profile(name);
+          const fileSaved = AWSCredentials.save_file();
+          this.log(
+            fileSaved === 1
+              ? "Account successfully removed"
+              : "There were an issue removing your new account"
+          );
+          break;
+        }
+        throw new Error(
+          `The flags name is required, use the flag --help for documentation`
+        );
+      }
+      case "show": {
+        this.log(AWSCredentials.serialize_credentials());
+        break;
+      }
+      default: {
+        break;
+      }
+    }
+  }
+}
+
+AWSAccounts.description = `Describe the command here
+...
+Allow to adminsitrate your AWS accounts
+`;
+
+AWSAccounts.flags = {
+  name: flags.string({
+    char: "n",
+    description: "Name of the AWS account",
```

Comment:
just for comment, you can set the default values here instead of using ` || undefined` in the code

Reply:
@FlavioAandres I made the changes to set a default on the flags object for each flag.